MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ElementSerializerT.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate
 {
     abstract class ElementSerializer<T> : ContentTypeSerializer<T>
     {
-        private static readonly char [] _seperators = { ' ', '\t' };
+        private static readonly char [] _seperators = { ' ', '\t', '\n' };
```

Comment:
I think this is safe.  If all our tests pass with it then at least for now it should be fine.